mods/cnc/maps/nod06a/nod06a.lua

```diff
@@ -0,0 +1,235 @@
+Win1CellTriggerActivator = { CPos.New(59,43) }
+Win2CellTriggerActivator = { CPos.New(54,58), CPos.New(53,58), CPos.New(52,58), CPos.New(54,57), CPos.New(53,57), CPos.New(52,57), CPos.New(54,56), CPos.New(53,56), CPos.New(52,56), CPos.New(54,55), CPos.New(53,55), CPos.New(52,55) }
+
+Grd2ActorTriggerActivator = { Guard1, Guard2, Guard3 }
+Atk1ActorTriggerActivator = { Atk1Activator1, Atk1Activator2 }
+Atk2ActorTriggerActivator = { Atk2Activator1, Atk2Activator2 }
+Chn1ActorTriggerActivator = { Chn1Activator1, Chn1Activator2, Chn1Activator3, Chn1Activator4, Chn1Activator5 }
+Chn2ActorTriggerActivator = { Chn2Activator1, Chn2Activator2, Chn2Activator3 }
+Obj2ActorTriggerActivator = { Chn1Activator1, Chn1Activator2, Chn1Activator3, Chn1Activator4, Chn1Activator5, Chn2Activator1, Chn2Activator2, Chn2Activator3, Atk3Activator }
+
+Chn1Waypoints = { ChnEnrty.Location, waypoint5.Location }
+Chn2Waypoints = { ChnEnrty.Location, waypoint6.Location }
+Gdi3Waypoints = { waypoint1, waypoint3, waypoint7, waypoint8, waypoint9 }
+Gdi4Waypoints = { waypoint4, waypoint10, waypoint9, waypoint11, waypoint9, waypoint10 }
+Gdi5Waypoints = { waypoint1, waypoint4 }
+Gdi6Waypoints = { waypoint2, waypoints3 }
+
+Grd1TriggerFunctionTime = DateTime.Seconds(3)
+
+Grd1TriggerFunction = function()
+	MyActors = Utils.Take(2, GDI.GetActorsByType('mtnk'))
+	Utils.Do(MyActors, function(actor)
+		MovementAndHunt(actor, Gdi3Waypoints)
+	end)
+end
+
+Grd2TriggerFunction = function()
+	if not Grd2Switch then
+		for type, count in pairs({ ['e1'] = 2, ['e2'] = 1, ['jeep'] = 1 }) do
+			MyActors = Utils.Take(count, GDI.GetActorsByType(type))
+			Utils.Do(MyActors, function(actor)
+				MovementAndHunt(actor, Gdi4Waypoints)
+			end)
+		end
+		Grd2Swicth = true
+	end
+end
+
+Atk1TriggerFunction = function()
+	if not Atk1Switch then
+		for type, count in pairs({ ['e1'] = 3, ['e2'] = 3, ['jeep'] = 1 }) do
+			MyActors = Utils.Take(count, GDI.GetActorsByType(type))
+			Utils.Do(MyActors, function(actor)
+				MovementAndHunt(actor, Gdi5Waypoints)
+			end)
+		end
+		Atk1Switch = true
+	end
+end
+
+Atk2TriggerFunction = function()
+	if not Atk2Switch then
+		for type, count in pairs({ ['mtnk'] = 1, ['jeep'] = 1 }) do
+			MyActors = Utils.Take(count, GDI.GetActorsByType(type))
+			Utils.Do(MyActors, function(actor)
+				MovementAndHunt(actor, Gdi6Waypoints)
+			end)
+		end
+		Atk2Switch = true
+	end
+end
+
+Atk3TriggerFunction = function()
+	if not Atk3Switch then
+		local list = Nod.GetGroundAttackers()
+		AirstrikeProxy = Actor.Create("airstrike.proxy", true, { Owner = GDI })
+		AirstrikeProxy.SendAirstrike(list[DateTime.GameTime % #list + 1].CenterPosition, false, 256 - 28)
+		AirstrikeProxy.Destroy()
+		Atk3Switch = true
+	end
+end
+
+Chn1TriggerFunction = function()
+	Reinforcements.ReinforceWithTransport(GDI, 'tran', Chn1Units, Chn1Waypoints, nil,
```

Comment:
You should change this to
```lua
Chn1TriggerFunction = function()
	local cargo = Reinforcements.ReinforceWithTransport(GDI, 'tran', Chn1Units, Chn1Waypoints, { waypoint14.Location })[2]
	Utils.Do(cargo, function(actor)
		IdleHunt(actor)
	end)
end
```
to avoid bogus unloading.